src/uvw/handle.hpp

```diff
@@ -243,6 +243,8 @@ class handle: public resource<T, U, close_event, E...> {
         uv_fileno(as_uv_handle(), &fd);
         return fd;
     }
+
+    std::function<void(uv_buf_t *, std::size_t, T &)> allocator = details::default_allocator<T>;
```

Comment:
I'm definitely not a fan of public data members that expose implementation details. Like, at all.
Put aside that one can easily _reset_ the function and makes everything fail abruptly now. No good.

Reply:
Ok, I admit it was a bad design.